frontends/web/src/routes/bitsurance/dashboard.module.css

```diff
@@ -11,7 +11,6 @@
 .accountsContainer {
   display: flex;
   flex-direction: column;
-  gap: var(--space-half);
```

Comment:
I noticed that the space between the accounts container doesn't show up on qt-windows. Possibly not supported? I used the `margin-botton` in .row instead. 
cc @thisconnect